controllers/main.js

```diff
@@ -1,7 +1,9 @@
 const { getPixelatedImage } = require("../middleware");
 
+const style = require("../public/javascripts/extraStyles.js");
```

Comment:
Refer to this comment: https://github.com/Leedukas/pixel-thing/pull/8/commits/91dce2cba82f659feadec6c7554af81cf9b7f036#r1160728518